config/config.go

```diff
@@ -315,12 +332,10 @@ type ScrapeConfig struct {
 	BearerToken string `yaml:"bearer_token,omitempty"`
 	// The bearer token file for the targets.
 	BearerTokenFile string `yaml:"bearer_token_file,omitempty"`
-	// The ca cert to use for the targets.
-	CACert string `yaml:"ca_cert,omitempty"`
-	// The client cert authentication credentials for the targets.
-	ClientCert *ClientCert `yaml:"client_cert,omitempty"`
 	// HTTP proxy server to use to connect to the targets.
 	ProxyURL URL `yaml:"proxy_url,omitempty"`
+	// Inlined TLSConfig.
+	TLSConfig TLSConfig `yaml:",inline"`
```

Comment:
Yeah, :+1: for having them not inlined, but in their own subsection. Already things like `KeyFile` and `Insecure` could mean anything if it's not clear that they are about TLS.